src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__photo"
+        src="images/imac.jpeg"
+        alt="imac"
+      >
+      <p class="card__product-name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="card__product-id">
+        Код товара: 195434
+      </p>
```

Comment:
```suggestion
      >
      
      <p class="card__product-name">
        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
      </p>
      
      <p class="card__product-id">
        Код товара: 195434
      </p>
```
Add empty lines between multiline sibling blocks of HTML